Dockerfile

```diff
@@ -36,11 +40,19 @@ RUN apk add --no-cache \
     # needed so non root users can read content of the bundle
     && chmod -R a+r /usr/local/bundle
 
+RUN mkdir -p $TOOLS_HOME/bin && \
+    cd $TOOLS_HOME/bin && \
+    curl -O https://dl.google.com/go/go1.11.2.src.tar.gz && \
+    tar -zxf go1.11.2.src.tar.gz && \
+    rm go1.11.2.src.tar.gz && \
+    cd go/src && \
+    ./make.bash
```

Comment:
Just to confirm; Why do we need this as well as `apk add go`? 👀

Reply:
Everything above line 51 is done in a temporary container during build.

Go above is added to the env builder and `go` is used to build `go` versions.  This builds the specific version of `go` toolchain that is then copied into the final container volume.

The published final container has the specified go version and location this way.